tests/budget_accounting_test.py

```diff
@@ -97,13 +97,22 @@ class ComputeBudgetTestCase:
             mechanisms: []
 
         testcases = [
+            ComputeBudgetTestCase(
+                name="generic",
+                epsilon=0.19348133991361996,
```

Comment:
@dvadym I am not sure why this test results in returning `math.inf`in line https://github.com/google/differential-privacy/blob/main/python/dp_accounting/privacy_loss_distribution.py#L596

It does not look like it should be catastrophic event 😨 ?

Reply:
I left comments to fix this

Reply:
Thank you!! My mistake definitely broke this. It makes sense and works in the tests now.

Thank you for spotting my error! 🚀